src/components/calcite-block-section/calcite-block-section.tsx

```diff
@@ -142,7 +142,7 @@ export class CalciteBlockSection {
       );
 
     return (
-      <section aria-expanded={open.toString()} class={dir ? CSS_UTILITY.rtl : null}>
+      <section aria-expanded={open.toString()} class={dir === "rtl" ? CSS_UTILITY.rtl : null}>
```

Comment:
Since this is dynamic, can you use a 'classes object' instead?

```tsx
<section ... class={{
  [CSS_UTILITY.rtl]: dir === "rtl"
}}>
```

Reply:
I fuhgot.

Reply:
Thanks @jcfranco! Updated.